sdk/search/azure-search/src/test/java/com/azure/search/SearchServiceSyncTests.java

```diff
@@ -26,4 +31,16 @@ public void getServiceStatsReturnsCorrectDefinitionWithResponse() {
         ServiceStatistics serviceStatistics = serviceClient.getServiceStatisticsWithResponse(generateRequestOptions(), Context.NONE).getValue();
         assertReflectionEquals(serviceStatistics, getExpectedServiceStatistics(), IGNORE_DEFAULTS);
     }
+
+    @Test
+    public void getServiceStatsReturnsRequestId() {
+        SearchServiceClient serviceClient = getSearchServiceClientBuilder().buildClient();
+
+        UUID expectedUuid = UUID.randomUUID();
+        RequestOptions requestOptions = new RequestOptions().setClientRequestId(expectedUuid);
+        Response<ServiceStatistics> response = serviceClient.getServiceStatisticsWithResponse(requestOptions, Context.NONE);
+
+        assertEquals(expectedUuid.toString(), response.getHeaders().getValue("client-request-id"));
```

Comment:
@alzimmermsft In the next month or so, we'll be deploying a service update that will set both `client-request-id` and `x-ms-client-request-id` on the response if either `return-client-request-id` or `x-ms-return-client-request-id` is specified.